pytorch_lightning/core/saving.py

```diff
@@ -368,7 +368,7 @@ def save_hparams_to_yaml(config_yaml, hparams: Union[dict, Namespace]) -> None:
         for v in hparams.values():
             if OmegaConf.is_config(v):
                 with fs.open(config_yaml, "w", encoding="utf-8") as fp:
-                    OmegaConf.save(OmegaConf.create(hparams), fp, resolve=True)
+                    OmegaConf.save(OmegaConf.create(v), fp, resolve=True)
                 return
```

Comment:
This snippet does not make a whole lot of sense to me.
It seems to be iterating the hparams, and saving the first value that is an OmegConf config, and ignore the rest?

Reply:
@Borda Any idea ?